app/Controllers/CollaneController.php

```
@@ -350,21 +287,32 @@ public function updateOrder(Request $request, Response $response, mysqli $db): R
+            $exists = (bool) $checkStmt->get_result()->fetch_assoc();
+            $checkStmt->close();
+            if (!$exists) {
+                $response->getBody()->write(json_encode(['error' => true, 'message' => __('Libro non in collana')], JSON_UNESCAPED_UNICODE));
+                return $response->withStatus(404)->withHeader('Content-Type', 'application/json');
+            }
         }
-        $stmt->bind_param('si', $numero, $bookId);
-        $ok = $stmt->execute();
-        $stmt->close();
+
+        $ok = (new SeriesRepository($db))->updatePrimaryOrder($bookId, $numero === '' ? null : $numero);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)